src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split(';');
+  const cssStyles = {};
+
+  for (let i = 0; i < lines.length; i++) {
+    const line = lines[i].trim();
+
+    if (!line) {
+      continue;
+    }
+
+    const lineSplitted = line.split(':');
+    const [cssProperty, cssValue] = lineSplitted.map((el) => el.trim());
+
+    if (cssProperty && cssValue) {
+      cssStyles[cssProperty] = cssValue;
```

Comment:
You're correctly checking if both `cssProperty` and `cssValue` are not empty before adding them to the `cssStyles` object. This is a good practice to avoid adding invalid entries.